contrib_bots/lib/myanimelist/mal.py

```diff
@@ -0,0 +1,69 @@
+import requests
+from bs4 import BeautifulSoup
+import re
+# See readme.md for instructions on running this code.
+malSearchUrl = "https://myanimelist.net/search/all?q="
+
+class LinksHandler(object):
+    def search(self, query):
+        if query != '':
+            searchPage = BeautifulSoup(requests.get(malSearchUrl + query.replace(' ', '%20')).content, 'html.parser')
+            try:
+                response = searchPage.findAll('div', {'class': 'list di-t w100'})[0]
+                details = response.findAll('div', {'class': 'pt8 fs10 lh14 fn-grey4'})[0].get_text()
+
+                returnDict = {
+                    'link': response.findAll('a', {'class': 'hoverinfo_trigger fw-b fl-l'})[0]['href'],
+                    'score': 'Scored ' + re.findall('Scored\ (.*?)\n', details)[0],
+                    'members': re.findall('(.*?)\ members', details)[0] + ' members',
+                    'type': re.findall('(.*?)\n', details)[1].replace('\u2028','')
+                }
+                print(re.findall('(.*?)\n', details))
+                print "YO IM HERE" + returnDict['type']
+                print(details)
+                return returnDict
+            except IndexError:
+                # will provide with no results
+                pass
```

Comment:
So this returns `None` when no results are found. Are you dealing with that when you are calling method?